api.md

```diff
@@ -1041,7 +1040,7 @@ WHERE hypertable = 'conditions';
 ```
 reschedules the reorder policy job for the `conditions` table so that it runs every two days.
 
-
+---
 ## Analytics [](analytics)
 
 ## first() [](first)
```

Comment:
Did you not put a divider here because `first` comes immediately after the `Analytics` header?